packages/devtools/web/styles.css

```diff
@@ -520,6 +520,8 @@ div.tabnav {
 .frame-timeline {
     position: relative;
     margin-top: 4px;
+    whitespace: pre;
+    overflow: scroll;
```

Comment:
DBC: this is scrolling within a frame on overflow? Perhaps use `hidden` instead?